packages/design-system/src/components/Dialog/Dialog.tsx

```diff
@@ -1,194 +1,19 @@
-   * The best way to add some vertical displacement to the dialog is to add
-   * top & bottom padding to the underlay.
-   * This is illustrated in the demo examples.
-   */
-  underlayStyle?: React.CSSProperties | undefined;
-
-  /**
-   * Apply a class to the underlay in order to custom-style it.
-   * This module does apply various inline styles, though, so be aware that
-   * overriding some styles might be difficult. If, for example, you want
-   * to change the underlay's color, you should probably use the
-   * `underlayColor` prop instead of a class.
-   * If you would rather control all CSS, see `includeDefaultStyles`.
-   */
-  underlayClass?: string | undefined;
-
-  /**
-   * By default, a click on the underlay will exit the modal.
-   * Pass `false`, and clicking on the underlay will do nothing.
-   */
-  underlayClickExits?: boolean | undefined;
-
-  /**
-   * By default, the Escape key exits the modal. Pass `false`, and it won't.
-   */
-  escapeExits?: boolean | undefined;
-
-  /**
-   * If you want to change the underlay's color, you can
-   * do that with this prop. If `false`, no background color will be
-   * applied with inline styles. Presumably you will apply then
-   * yourself via an `underlayClass`.
-   *
-   * Default: rgba(0,0,0,0.5)
-   */
-  underlayColor?: string | false | undefined;
-
-  /**
-   * If `true`, the modal's contents will be vertically (as well as horizontally) centered.
-   */
-  verticallyCenter?: boolean | undefined;
-
-  /**
-   * This function is called in the modal's `componentDidMount()` lifecycle method.
-   * You can use it to do whatever diverse and sundry things you feel like
-   * doing after the modal activates.
-   */
-  onEnter?(): void;
-
-  /**
-   * This function needs to handles the state change of exiting (or deactivating) the modal.
-   * Maybe it's just a wrapper around `setState()`; or maybe you use some more involved
-   * Flux-inspired state management — whatever the case, this module leaves the state
-   * management up to you instead of making assumptions.
-   * That also makes it easier to create your own "close modal" buttons; because you
-   * have the function that closes the modal right there, written by you, at your disposal.
-   */
-  onExit?(event: React.MouseEvent | React.KeyboardEvent): void;
-
-  /**
-   * If true, the modal dialog's focus trap will be paused.
-   * You won't typically need to use this prop. It used to be that the typical reason for pausing a focus trap was to enable nested focus traps;
-   * but as of focus-trap v4, the pausing and unpausing of hierachical traps is handled automatically.
-   */
-  focusTrapPaused?: boolean | undefined;
-
-  /**
-   * Customize properties of the focusTrapOptions prop that is passed to the modal dialog's focus trap.
-   * For example, you can use this prop if you need better control of where focus is returned.
-   */
-  focusTrapOptions?: Record<string, unknown> | undefined;
+declare const window: any;
```

Comment:
I had to add this line to skirt some TS errors. TS wasn't reading `HTMLDialogElement` as a valid type of window (even though it should be?). Maybe our types are outdated?